### PR TITLE
fix: use context.env instead of process.env

### PR DESCRIPTION
**Pull request by fengmk2**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

**Comment by fengmk2** [comment]
依赖 https://github.com/node-modules/common-bin/pull/12 合并发布

File: lib/cmd/dev.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ class DevCommand extends Command {
     const devArgs = yield this.formatArgs(context);
     const options = {
       execArgv: context.execArgv,
-      env: Object.assign({ NODE_ENV: 'development' }, process.env),
+      env: Object.assign({ NODE_ENV: 'development' }, context.env),
```
Comment (fengmk2) [inline lib/cmd/dev.js]: 方便上层 bin 覆盖 env

Comment (atian25) [inline lib/cmd/dev.js]: 内网版那里有可能也要跟着改

Comment (fengmk2) [inline lib/cmd/dev.js]: 不需要，内网是继承的

File: package.json [hunk 2]
```diff
@@ -10,14 +10,14 @@
...
-    "debug": "^2.6.4",
-    "detect-port": "^1.1.1",
-    "egg-utils": "^2.1.0",
+    "common-bin": "^2.4.0",
+    "debug": "^2.6.8",
+    "detect-port": "^1.1.3",
+    "egg-utils": "^2.2.0",
     "globby": "^6.1.0",
     "intelli-espower-loader": "^1.0.1",
-    "istanbul": "^1.1.0-alpha.1",
```
Comment (fengmk2) [inline package.json]: 这个我还原一下

**Comment by fengmk2** [comment]
3.4.2

**Comment by popomore** [comment]
+1

**Pull request merged**